index.html

```diff
@@ -1,8 +1,89 @@
 <!DOCTYPE html>
-<html>
-   <head>
-   </head>
-   <body>
-      <!-- Write your HTML in this file! -->
-   </body>
-</html>
\ No newline at end of file
+<html lang="en">
+  <head>
+
+  <meta charset="utf-8">
+  
+  <title>My Website</title>
+      <link rel="stylesheet" href="styles.css" type="text/css">
+  </head>
+    
+<div>
+   <header id="m_mainHeading">
+
+      <h1>My Sweet Garden<h1>
+   </header>
+  
+<p id="m_testP" class="m_funParagraph">
+  I have a small sweet garden and plant a few vegetables. I always love planting.My kids love vegetables. Every morning they check  plants and pick them up.
+  
+   </p>
+
+
+    <nav class="menu">
+      <ul>
+      
+        <li><a href="#Tomates">Tomates</a></li>
+        <li><a href="#Cucember">Cucember</a></li>
+        <li><a href="#Peppers">Peppers</a></li> 
+        </ul> 
+
+          </nav>
+  
+      <header>
+        <h2>Tomates</h2>
+        </header>
+                <img
+            src="images/tomates.jpg"
```

Comment:
It looks like it's looking for an image in the images folder called tomates.jpg. There were no images checked in to the code, so it doesn't show up when I pull the code down.